diagrams/saas/chat.py

```diff
@@ -8,10 +8,22 @@ class _Chat(_Saas):
     _icon_dir = "resources/saas/chat"
 
 
+class Mattermost(_Chat):
+    _icon = "mattermost.png"
+
+
+class Rocketchat(_Chat):
```

Comment:
It should be `class RocketChat`. I left a similar request on your previous PRs.

Reply:
Done. Thanks.